tck/src/main/java/org/reactivestreams/tck/support/PublisherVerificationRules.java

```diff
@@ -297,6 +297,18 @@
    */
   void optional_spec111_maySupportMultiSubscribe() throws Throwable;
   /**
+   * Ask for a single-element {@code Publisher} and subscribes to it twice.
+   * Each {@code Subscriber} requests for 1 element and checks if onNext or onComplete signals was received.
```

Comment:
This is a bid on the edge; imagine a Publisher that does not know if it can signal or not (if it is completed or not) until someone signals demand to it. Specifically, we request 1, it signals, we *know* wont signal more elements but the Publisher does not know that - e.g. it has to check on some external source if it can pull from it, and only does so when demand arrives - completely legal impl. So such lazy impl, may need to get an additional request(1) to actually trigger the completion.

So this test should allow both, very lazy, and eager Publisher impls to pass. Easiest is to request 2 or request 1 and then 1.